# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/APZCTreeManager.h
# Commit: 2e296c018c3b
# Full Hash: 2e296c018c3b62984150b42cec757af680d18695
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2021-02-02 03:35:00
# Regressor Bug: 1687067
# File Overlap Count: 1
# Description:
#   Bug 1687067 - Build OverscrollHandoffChain by walking up hit testing tree instead of APZC tree. r=botond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D103601
# ==============================================================================

diff -r a32d58612fb7 -r 2e296c018c3b gfx/layers/apz/src/APZCTreeManager.h
--- a/gfx/layers/apz/src/APZCTreeManager.h	Mon Feb 01 23:23:42 2021 +0000
+++ b/gfx/layers/apz/src/APZCTreeManager.h	Mon Feb 01 23:25:17 2021 +0000
@@ -621,6 +621,8 @@
                                      const ScrollableLayerGuid& aGuid,
                                      GuidComparator aComparator);
   AsyncPanZoomController* GetTargetApzcForNode(HitTestingTreeNode* aNode);
+  AsyncPanZoomController* FindHandoffParent(
+      const AsyncPanZoomController* aApzc);
   HitTestResult GetAPZCAtPoint(const ScreenPoint& aHitTestPoint,
                                const RecursiveMutexAutoLock& aProofOfTreeLock);
   HitTestResult GetAPZCAtPointWR(